lib/components/form/date-time-preview.js

```diff
@@ -63,19 +68,22 @@ class DateTimePreview extends Component {
       </div>
     )
 
-    const button = (
-      <div className='button-container'>
+    const button = hideButton
```

Comment:
`button = !hideButton && (<div... >)`

Reply:
Going to leave this how it is. This is just a matter of preference and we use both throughout the app.